singer/catalog.py

```diff
@@ -11,7 +11,7 @@ class CatalogEntry(object):
     def __init__(self, tap_stream_id=None, stream=None,
                  key_properties=None, schema=None, replication_key=None,
                  is_view=None, database=None, table=None, row_count=None,
-                 stream_alias=None, metadata=None):
+                 stream_alias=None, metadata=None, bookmark_properties=None):
```

Comment:
You should not need to change the catalog at all. You can revert all the changes on this file.